tests/manual/autoformat.md

```diff
@@ -0,0 +1,8 @@
+@bender-ui: collapsed
+@bender-tags: autoformat
+
+## Autoformat
+
+1. Type `#` and press space to replace current paragraph with the heading.
+2. Type `*` and press space to replace current paragraph with list item.
+3. Type number from the range `1`-`3` to replace current paragraph with numbered list item.
```

Comment:
There should be a TC checking integration with undo. There were some problems with implementing this correctly, so a manual test case(s) for that would be nice.